src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
```

Comment:
Same here the div that is wrapping the Link component is redundant. You can pass css className props to Link component, if you want to control the style of those elements.